app/src/components/market/common/position_selection_box/index.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { formatBigNumber, formatNumber, isDust, reverseArray } from '../../../../util/tools'
+import { BalanceItem } from '../../../../util/types'
+import { RadioInput } from '../../../common'
+
+const PositionSelectionBoxWrapper = styled.div`
+  height: 88px;
+  border: ${props => props.theme.borders.borderLineDisabled};
+  border-radius: ${props => props.theme.cards.borderRadius};
+  margin-bottom: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 16px 20px;
```

Comment:
This values could be imported from theme theres paddingVertical and paddingHorizontal value...

Reply:
I actually think we shouldn't import these values because they're not consistent between vertical and horizontal. For example, we have `cards.paddingHorizontal: '20px'` and `cards.paddingVertical: '12px'`, which is inconsistent with ours at `16px 20px` meaning it probably shouldn't be classified as a card. If we were to create another element with similar dimensions, I would say it'd be good to create a whole `theme.positionBox` object. Overall, the theme file doesn't have a consistent system atm which should really be fixed